frontend/src/scenes/feature-flags/featureFlagsLogic.ts

```diff
@@ -41,36 +56,40 @@ export const featureFlagsLogic = kea<featureFlagsLogicType>([
     actions({
         updateFlag: (flag: FeatureFlagType) => ({ flag }),
         deleteFlag: (id: number) => ({ id }),
-        setSearchTerm: (searchTerm: string) => ({ searchTerm }),
         setActiveTab: (tabKey: FeatureFlagsTab) => ({ tabKey }),
         setFeatureFlagsFilters: (filters: Partial<FeatureFlagsFilters>, replace?: boolean) => ({ filters, replace }),
         closeEnrichAnalyticsNotice: true,
     }),
     loaders(({ values }) => ({
         featureFlags: {
-            __default: [] as FeatureFlagType[],
+            __default: { results: [], count: 0, filters: null, offset: 0 } as FeatureFlagsResult,
             loadFeatureFlags: async () => {
-                const response = await api.get(`api/projects/${values.currentTeamId}/feature_flags/?limit=300`)
```

Comment:
oh, interesting, so it looks like we were capping at 300 just for some arbitrary reason.